tests/unit_tests/test_universe.py

```diff
@@ -31,6 +31,42 @@ def test_basic():
     u.clear_cells()
     assert not set(u.cells)
 
+def test_contains_cell():
+
+    c1 = openmc.Cell()
+    c2 = openmc.Cell()
+    c3 = openmc.Cell()
+    c3 = openmc.Cell()
+    c4 = openmc.Cell()
+
+    u0 = openmc.Universe(name='coldplay', cells=(c1, c2, c3))
+
+
+    assert(c1 in u0)
+    assert(c2 in u0)
+    assert(c2 in u0)
+    assert not (c4 in u0)
+    assert not ('hi' in u0)
+
+
+    c5 = openmc.Cell()
+    u5 = openmc.Universe(cells=[c5])
+    c6 = openmc.Cell()
+    u6 = openmc.Universe(cells=[c6])
+    c7 = openmc.Cell(fill=u5)
+    c8= openmc.Cell(fill=u6)
+    u7 = openmc.DAGMCUniverse(filename="")
+    c9 = openmc.Cell(fill=u7)
+    u_composed = openmc.Universe(cells=[c7, c8, c9])
+
+    assert not c1 in u_composed
+    assert c5 in u_composed
+    assert c6 in u_composed 
+    assert c8 in u_composed
+    assert c9 in u_composed
+    assert not c4 in u_composed 
+
+    assert not 'hi' in u_composed
```

Comment:
I think this should be: 

``` python
with pytest.raises(TypeError): #error type might be different I'm just assuming
    'hi' in u_composed
```

Reply:
@MicahGale I'm not sure I understand. I altered the lines of test_universe.py to:
![error_test_uni](https://github.com/openmc-dev/openmc/assets/3958502/5260ff91-96b9-47b5-89c5-ce3f57885795)

This indeed triggers the TypeError exception from the logic of universe.py to
![universe_contains_TypeError](https://github.com/openmc-dev/openmc/assets/3958502/d7875bb9-a9f3-4e7b-a837-2f647c37d117)

This produced the following lone error when I run pytest test_universe.py:
![TypeError1](https://github.com/openmc-dev/openmc/assets/3958502/a44f9053-6c0d-4baf-a8c8-ff807f7d94b4)

Is there example of  raising TypeError within the in another test script that I could look at? I assume I'm missing some syntax to.... contain the TypeError exception within the assert statement (analougous to a try-catch block)

Reply:
@MicahGale never mind. I figured it out